### DISCUSSION TITLE
Field displayed on the dropdown when attach

### DISCUSSION THREAD

**Discussion by vianeyHA** [avo-hq/avo#3349] [Help] [answered]
How can I define what field will be displayed for a "has_many" field when clicking the "attach" button?
in my project, it takes by default the ID  but in [this example](https://github.com/avo-hq/main.avodemo.com/blob/main/app/avo/resources/post.rb) displays a description

Example:
![image](https://github.com/user-attachments/assets/5b00f998-3312-4f68-a7d8-e466fd8f407d)

**Comment by Paul-Bob** [comment, marked as answer]
Hi @vianeyHA

Avo renders the label for the select options on the attach modal using the [`self.title`](https://docs.avohq.io/3.0/resources.html#self.title) from the target resource.

The example that you shared uses [`self.title = :tiny_name`](https://github.com/avo-hq/main.avodemo.com/blob/main/app/avo/resources/comment.rb#L2) from the target (`Avo::Resources::Comment`) resource.

[`tiny_name`](https://github.com/avo-hq/main.avodemo.com/blob/main/app/models/comment.rb#L19) is a method (but could be directly an attribute) on the `Comment` model.
